mmcv/cnn/bricks/conv_module.py

```diff
@@ -105,7 +105,9 @@ def __init__(self,
         self.with_bias = bias
 
         if self.with_norm and self.with_bias:
-            warnings.warn('ConvModule has norm and bias at the same time')
+            if self.conv_cfg.get('type').startswith('BN'):
```

Comment:
Should we replace the `self.conv_cfg` with `self.norm_cfg` ?
In addition, if the user extends from BN, the condition is not enough robust.
```python
NORM_LAYERS.register_module()
class DummyBN(bb.BatchNorm2d):
     pass
```

Reply:
Sorry, will fix the spelling mistake asap. You are right  about the robustness problem. If possible, the isinstance method would be a better choice, but I could  not find the right place.

Reply:
Maybe we can use `issublclass()`
```python
obj_type = self.norm_cfg.get('type')
obj_cls = registry.get(obj_type)
if any(issubclass(obj_cls, norm_cls) for norm_cls in [nn.BatchNorm1d, ...]):
    warnings.warn('ConvModule ...')
```

Reply:
Got it. This is actually beyond my knowledge, thanks.